capstone/src/main/java/com/google/step/servlets/LoginServlet.java

```diff
@@ -55,14 +55,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       login.put("LoggedIn", true);
       login.put("Email", userEmail);
-      login.put("logOutURL", logoutURL);
+      login.put("LogOutURL", logoutURL);
```

Comment:
Is there a reason you changed this to be capital? 

The google style guide for JSON says property names should be camel cased:
https://google.github.io/styleguide/jsoncstyleguide.xml#Property_Name_Format

Reply:
Sorry fixed!